core/dev/openhim/console-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: console
+  name: console
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: console
+  template:
+    metadata:
+      annotations:
+      labels:
+        app: console
+    spec:
+      containers:
+        - image: jembi/openhim-console:1.13
+          name: openhim-console
+          ports:
+            - containerPort: 80
+          volumeMounts:
+            - name: console-config
+              mountPath: /usr/share/nginx/html/config/default.json
+              subPath: console-config.json
+          env:
```

Comment:
@bradsawadye I don't think this ENV var is needed? Does console care about the `NODE_ENV`?